options/locale/locale_fr-FR.ini

```diff
@@ -921,10 +921,12 @@ unstar=Retirer des favoris
 star=Ajouter aux favoris
 fork=Bifurcation
 download_archive=Télécharger ce dépôt
+more_actions=Plus d'Actions
```

Comment:
will be overwritten!

Reply:
plese dont change any locale files exept the en-US ...
after a pull got merged, you can translate it into your language at https://crowdin.com/project/gitea